admin/js/monk-admin.js

```diff
@@ -87,6 +84,10 @@
 			var encoded_url = $( 'select[name="monk-term-translation"]' ).val();
 			window.location.replace( encoded_url );
 		});
-	});
 
+		$( document ).on( 'hover', 'td.column-languages', function() {
+			$( this ).children( '.hide-if-js, .monk-hide' ).slideToggle();
```

Comment:
Remove `.slideToggle()` and add `display: block;` property on `.monk-show` css class